src/app/my/canvassassignments/page.tsx

```diff
@@ -0,0 +1,21 @@
+import { headers } from 'next/headers';
+import { notFound } from 'next/navigation';
+
+import BackendApiClient from 'core/api/client/BackendApiClient';
+import MyCanvassAssignmentsPage from 'features/areas/components/MyCanvassAssignmentsPage';
+import { ZetkinOrganization } from 'utils/types/zetkin';
+
+export default async function Page() {
+  const headersList = headers();
+  const headersEntries = headersList.entries();
+  const headersObject = Object.fromEntries(headersEntries);
+  const apiClient = new BackendApiClient(headersObject);
+
+  try {
+    await apiClient.get<ZetkinOrganization>(`/api/users/me`);
```

Comment:
I honestly don't know if this line of code does anything, I just improvised it based on a similar pattern I saw in another file 🤷 I guess the idea I think i might be achieving is to check if that api call works out, and if not, return 404, but not sure this is right or neccessary

Reply:
It's not a perfect solution, but it's not bad and it does return a 404 if you're not logged in so in that sense it is doing what you intended.